src/components/RenderElement.tsx

```diff
@@ -37,31 +38,38 @@ type RenderElementProps = {
 const RenderElement: FC<RenderElementProps> = (props) => {
   if (!props.element) {
     return (
-      <p className={`text-balance font-sans text-xl text-gray ${props.errorMessageClassName ?? ""}`}>
-        Missing or invalid element codename{" "}
+      <KontentComponentErrorMessage errorMessageClassName={props.errorMessageClassName}>
+        Missing element with the codename{"  "}
         <b>
           <i>{props.elementCodename}</i>
-        </b>. To learn more about codenames refer to the{" "}
+        </b>. Ensure that your{" "}
+        <a
+          href="https://kontent.ai/learn/try-kontent-ai/build-the-foundation/create-a-landing-page-structure#a-create-a-landing-page-content-type"
+          className="underline text-blue-600 hover:text-blue-800 visited:text-purple-600"
+        >
+          Landing Page content type
+        </a>{" "}
+        contains elements with elements names and{" "}
```

Comment:
i would leave just `contains elements with names and codenames`. if you insist on leaving the duplicit elements in there, it should be `contains elements with element names and codenames`